src/Css.elm

```diff
@@ -1037,6 +1126,22 @@ n num =
   }
 
 
+w : Int -> NumberedWeight
+w num =
+  let
+    validWeight nm = List.member nm <| List.map (\n -> n * 100) [1..9]
+  in
+    if validWeight num then
+      { value = numberToString num, fontWeight = Compatible }
+    else
+      if num < 100 then
+        { value = "100", fontWeight = Compatible }
+      else if num > 900 then
+        { value = "900", fontWeight = Compatible }
+      else
+        { value = numberToString <| round (num / 100) * 100
+        , fontWeight = Compatible }
+
```

Comment:
I think it would be more consistent to implement this using the existing `n` function (for bare numeric units) and a warning via `fontWeight` for invalid values.

For example:

`fontWeight (n 100)` yields:
```elm
{ key = "font-weight"
, value = "100"
, warnings = []
}
```

`fontWeight (n 22)` yields:
```elm
{ key = "font-weight",
, value = "22"
, warnings = [ "22 is not a valid font-weight value. Please see https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#Values" ]
}
```